vault/request_handling.go

```diff
@@ -1890,6 +1890,11 @@ func (c *Core) handleDelegatedAuth(ctx context.Context, origReq *logical.Request
 	authReq.MountAccessor = requestedAccessor
 	authReq.Path = path
 	authReq.Operation = logical.UpdateOperation
+
+	// filter out any response wrapping headers, for our embedded login request
+	delete(authReq.Headers, consts.WrapTTLHeaderName)
```

Comment:
If you go with the suggested name, you will need to use `textproto.CanonicalMIMEHeaderKey(consts.WrapTTLHeaderName)` here probably.